### PR TITLE
display non-existing user with email

### PR DESCRIPTION
**Pull request by bs-jokri**
- if user is stored in a field of a document, display user's email in details and edit view
- this way, in edit mode, the user is not deleted silently

### CODE REVIEW

**Comment by mcjaeger** [comment]
I understood this right that instead of "Freddie Mercury, ..." there will be "freddie.mercury@queen.co.uk"? in the fields in edit and detail view of some document?

**Comment by mcjaeger** [comment]
current screenshot of status in question

screen shot 2017-01-09 at 14 20 37

**Comment by mcjaeger** [comment]
OK, works for non-existing user, sorry, my confusion.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEmail.java [hunk 1]
```diff
@@ -33,12 +37,17 @@ public void doTag() throws JspException, IOException {
...
-            user = UserCacheHolder.getUserFromEmail(email);
-        } else {
-            email = "";
+            UserService.Iface client = new ThriftClients().makeUserClient();
+
+            if(client != null) {
+                try {
+                    user = client.getByEmail(email);
+                } catch(TException e) {
+                    log.warn("Exception occurred in User client.");
```
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEmail.java]: let's add exception message and parameter email used here ...

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEdit.java [hunk 2]
```diff
@@ -95,11 +97,13 @@ public int doStartTag() throws JspException {
                     if (!Strings.isNullOrEmpty(email))
                         user = client.getByEmail(email);
                 } catch (TException e) {
-                    user = null;
+                    log.warn("Exception occurred in User client.");
```
Comment (mcjaeger) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayUserEdit.java]: let's add exception message and parameter email used here ...

**Comment by maximilianhuber** [comment]
@mcjaeger the PR is now updated. Could you please update your review?

**Pull request merged**